scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -1251,6 +1237,27 @@ class KubernetesMonitor( ScalyrMonitor ):
 
     """
 
+    # A map of param to
+    ENVIRONMENT_AWARE_CONFIG_PARAMS = {
+        'container_check_interval': None,
+        'docker_max_parallel_stats': None,
+        'container_globs': None,
+        'report_container_metrics': None,
+        'report_k8s_metrics': None,
+        'k8s_ignore_pod_sandboxes': None,
+        'k8s_include_all_containers': None,
+        'k8s_parse_json': None,
+        'gather_k8s_pod_info': None,
+        # 'k8s_use_v2_attributes': None,
+        # 'k8s_use_v1_and_v2_attributes': None,
+        # The following params have deprecated (moved to global config)
+        # 'k8s_ignore_namespaces': None,
+        # 'k8s_api_url': None,
+        # 'k8s_verify_api_queries': None,
+        # 'k8s_cache_expiry_secs': None,
+        # 'k8s_cache_purge_secs': None,
+    }
+
```

Comment:
Explicitly comment-out params that should not be environment-aware (to indicate a conscious decision)